addons/cookoff/CfgVehicles.hpp

```diff
@@ -8,98 +8,24 @@ class CfgVehicles {
         sound = "ACE_CookOff";
     };
 
-    /*
-     * ACE_Cookoff_ammoLocation:
-     *   HitPoint classname that holds the magazine of the vehicle
-     * ACE_Cookoff_effectLocations[]:
-     *   Positions to spawn cookoff particle effects at
-     * ACE_Cookoff_effectOnTurret[]:
-     *   Positions placed on turret?
-     */
-
-    class Land;
-    class LandVehicle: Land {
-        GVAR(ammoLocation) = "HitTurret";
-        GVAR(effectLocations)[] = {{0,-0.2,0}};
-        GVAR(effectOnTurret)[]  = {0};
-    };
+    class LandVehicle;
     class Tank: LandVehicle {
-        // 2 hatches on the turret; fits most tanks
-        GVAR(effectLocations)[] = {{-0.6,-0.3,1.65}, {0.5,-0.3,1.65}};
-        GVAR(effectOnTurret)[]  = {1, 1};
+        GVAR(ammoLocation) = "HitTurret";
+        GVAR(selections)[] = {"poklop_gunner","poklop_commander"};
```

Comment:
Unfortunately these are not in the memory lod, so we can't attach to these points directly. It drastically simplifies the math though.